README.md

```diff
@@ -34,7 +44,9 @@ sudo pip3 install oval-graph
 sudo pip3 install inquirer
 
 ```
-#### Install from source
+
+### Installation from source
```

Comment:
I like this - a short installation howto followed by example usage and the display output are ideal contents of project's readme.